### PR TITLE
Next trip title

### PR DESCRIPTION
**Pull request by qiyundai**
The font files are in the assets.

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 require "rails/all"
 
+config.assets.paths << Rails.root.join('app', 'assets', 'fonts')
```
Comment (fredThem) [inline config/application.rb:5]: @qiyundai  This line is crashing the app
https://lewagon-alumni.slack.com/archives/C033YB2VC/p1618579293014200

**Review by fredThem — CHANGES_REQUESTED** [review]
Please see thread in comment

File: config/application.rb (hunk 1 above)
Comment (fredThem) [reply config/application.rb:5]: The thread mentions the line could be placed elsewhere. Just not sure if this is the right approche*

**Review by qiyundai — COMMENTED** [review]
This branch has since been updated and is ready for review.

**Comment by qiyundai** [comment]
Following with a screenshot

screencapture-localhost-3000-2021-04-16-16_48_52

**Pull request merged**